fastlane/Fastfile

```diff
@@ -132,7 +132,7 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
   #####################################################################################
   # new_hotfix_release
   # -----------------------------------------------------------------------------------
-  # This lane updates the release branch for a new hotix release. 
+  # This lane creates the release branch for a new hotix release. 
```

Comment:
While we're at it, can we fix this one too? 😄 
```suggestion
  # This lane creates the release branch for a new hotfix release. 
```

Reply:
Good catch!